docs/Changelog.md

```diff
@@ -2,7 +2,7 @@
 
 1.12.0 (Not released yet)
 -------------------------
-* [#4677](https://github.com/TouK/nussknacker/pull/4677) Added validation to SPeL string conversions (allow only constant that convert successfully)
+* [#4677](https://github.com/TouK/nussknacker/pull/4677) Added validation to SpEL string conversions (allow only constant that convert successfully)
```

Comment:
Added validation to SpEL string **literals** conversions.. - it's important that evaluated variables are still validated at the runtime.